mycli/main.py

```diff
@@ -790,10 +801,12 @@ def format_output(self, title, cur, headers, status, expanded=False,
                 formatted = self.formatter.format_output(
                     rows, headers, format_name='vertical')
 
-            output.append(formatted)
+            if isinstance(formatted, str) or isinstance(formatted, unicode):
```

Comment:
You might want to try adding `from mycli.encodingutils import text_type` to `main.py` so you can use that here.